Utilities/I18N.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.IO;
+
+namespace Utilities
+{
+    // Simple internationalization helper class
+    public class I18N
+    {
+        // default langCode, used if getString() fail with currentLang
+        private const string defaultLang = "en";
+        // current langCode
+        private static string currentLang;
+        // contains a I18NString for each internationalized string in langs.json
+        // key is the langCode, value is internationalized string
+        // to access a string use translations[stringCode][langCode]
+        private static Dictionary<string, I18NString> translations;
+        // contains each langs defined in langs.json
+        private static List<String> availableLangs;
+        // Can't use Utilities.Constants there because of cyclic dependecies
+        private static string dataDir = Environment.GetEnvironmentVariable("AppData") + @"\EDDI\";
```

Comment:
Let's see if we can find a way to break the cyclic dependency here.

Reply:
The only solution i've found was not to place something localized (or localizable) in Constants.cs file.
A "LocalizedConstants.cs" where constants like 'ENVIRONMENT_WITCH_SPACE' or 'VEHICLE_SHIP' will be placed may help. With this I think we can get rid of that cyclic dependency.

Reply:
Don’t worry, as soon as I migrate the data to RESX files the dependency will go away by itself.